### DISCUSSION TITLE
Contributor call: 28 May 2022

### DISCUSSION THREAD

**Discussion by l3xander** [freesewing/freesewing#2188] [Support] [answered]
You are all invited to the FreeSewing Contributor Call.
Please leave a comment here or [on Discord](https://discord.freesewing.org/) to add things to the agenda.
This issue will be updated after the call with notes and to-dos.

### About
FreeSewing's contributor call is an informal hangout/chat/meeting that is open to all FreeSewing contributors and users alike.
The call is organized/hosted by @tangerineshark and @lexander2002. 
The call does not have a fixed structure or agenda items. Everyone is free to suggest agenda items, ask questions, or bring ideas.

Details:
Location: https://discord.freesewing.org (meeting room: 🔊voice chat)
Date: 28 May 2022 4pm UTC, which can be found in your local time zone [here](https://savvytime.com/converter/utc-to-belgium-antwerpen-ca-los-angeles-il-chicago-ny-new-york-city-brazil-sao-paulo-united-kingdom-london-france-paris-south-africa-johannesburg-msd-india-mumbai-hkt-japan-tokyo-australia-sydney/apr-30-2022/4pm).
Or check the bot in Discord! #meetings
The meetings will always be at 6pm, Antwerp time.

### Agenda
- Jacqueline's feedback
- Default seam allowance in imperial units
- Should we add a `historical` category to our patterns? (we now have accessories, blocks, garments, and utilities)

> This discussion will be updated after the call with notes and to-dos.

**Comment by joostdecock** [comment]
I suspect the **Default seam allowance in imperial units** thing is a bit of a misnomer. Guess we'll figure it out during the discussion

**Comment by l3xander** [comment, marked as answer]
### Notes
- [New bug alert!](https://github.com/freesewing/freesewing/issues/2215) Seam allowance defaults to “Custom seam allowance” instead of “Standard seam allowance” when imperial units have been enabled in the account.
Thank you to @nataliasayang for finding and reporting the bug.
@joostdecock added that imperial units have always been a bit of a pain for FreeSewing.
- @starfetch brought up the question of, should we add a `historical` tag to patterns?
    - @starfetch explained that it won’t be its own separate category (after all, a pattern can be both historical and a garment), but a tag to filter with. That also brings the question, what other tags would we want to use? 
    It would make sense to add more general categories and tags in the future when we have more patterns, but it’s not a pressing issue now.
    - @joostdecock added that we’d need to add support for translating. We can add this, but we don’t have to worry about deciding on this now.
    - @bobgeorgethe3rd added that we’ll want to be careful with calling things ‘historical’ (many things could fall under that: sewing patterns or sewing techniques from previous time periods, or just ‘vintage-esque’). 
    We decided on “historically inspired”.
- @tangerineshark on new possible interviewees: AMJ/Anthony didn’t yet write back. For the next interview, @starfetch would be a good fit!
- @joostdecock did some work on the new website and brought awareness to an important point: now that different languages won’t be on separate sites, but rather all in one site where you can swap between languages, we will need ALL content in all the different languages.
    - That may sound obvious and not different to before, but for example, many blogposts haven’t been translated. Now untranslated blogposts simply don’t show up on the different language sites, but this is no longer an option on the new site where you swap between the languages. So blogposts will need a “not yet translated” marker or something similar.
    - And there’s also the problem if content is created in non-English first. (Like a user-made showcase.) The translations are based on “primary content”, which is English in our case. That primary content needs to exist first to build translations on top of.
- @joostdecock wants there to be a “featured” field on the new site, where ‘good quality’ showcase pictures will be shown. As a “glanceable overview” of a pattern.
    - @nataliasayang added that it would be helpful to use this to show how the pattern looks on different bodies.
    - @bobgeorgethe3rd added that we can do a designer-showcase, where the pattern is made by its designer, so we can see how it was intended to be made.

**Reply by bobgeorgethe3rd** [reply]
Just to clarify on what I meant on historical
Pure Historical: Using both historical drafting and historical method to construct.
Historically Inspired: Using historical drafting but not nesecarrily historical method to construct. (Also using Historical Pattern with Modern influences such as modern colours etc,)
Vintage-esque: Using modern method to create something that could be percived as "historical" but not actually using historical drafting.

**Comment by joostdecock** [comment]
Closing to increase visibility of active discussions
